File: app/src/main/java/com/fengniao/news/bean/TouTiaoNews.java
package com.fengniao.news.bean;

import com.google.gson.annotations.SerializedName;

public class TouTiaoNews {
    public String title;
    public String date;
    @SerializedName("author_name")
    public String author;
    public String url;
    @SerializedName("thumbnail_pic_s")
    public String picOne;
    @SerializedName("thumbnail_pic_s02")
    public String picTwo;
    @SerializedName("thumbnail_pic_s03")
    public String picThree;
}

File: app/src/main/java/com/fengniao/news/bean/TouTiaoNews.kt
package com.fengniao.news.bean

import com.google.gson.annotations.SerializedName

class TouTiaoNews {
    var title: String? = null
    var date: String? = null
    @SerializedName("author_name")
    var author: String? = null
    var url: String? = null
    @SerializedName("thumbnail_pic_s")
    var picOne: String? = null
    @SerializedName("thumbnail_pic_s02")
    var picTwo: String? = null
    @SerializedName("thumbnail_pic_s03")
    var picThree: String? = null
}
